xclim/indices/_agro.py

```diff
@@ -368,12 +368,13 @@ def rolling_drydays_events(
     Returns
     -------
     xarray.DataArray
-      The number of periods of minimum {window} days for each resampling period.
+      The number of dry periods of minimum {window} days for each resampling period.
     """
-    thresh = convert_units_to(thresh, pr, "hydro")
+    pram = rate2amount(convert_units_to(pr, "mm/day"))
```

Comment:
```suggestion
    pram = rate2amount(pr, out_units='mm')
```
I think this is more explicitly what you are looking for? At the same time, I don't think there's a real need to force units here?

Reply:
It's true, I don't need to force the units. 

I used convert_units_to() to update the units of pram in mm. This way I can use convert_units_to() on thresh to make sure it has the same units as pram.

I could update pram units separately.